web/src/gear_menu.js

```diff
@@ -136,6 +137,10 @@ export function initialize() {
         show_webathena: page_params.show_webathena,
     });
     $("#navbar-buttons").html(rendered_gear_menu);
+
+    $("body").on("click", "#settings-dropdown", () => {
+        popovers.hide_all();
+    });
```

Comment:
What's the story with this block? Do we not have an existing click handler to add this to, or does it not have `stopPropagation`? (Maybe it's a bootstrap click handler?)

Reply:
We do not have a click handler for gear_menu or settings-dropdown. We use `data-toggle="dropdown"` to open gear_menu.
```
```

Reply:
Removing call for `gear_menu` for now since `popovers.js` hasn't been converted to ts yet.